SORT/settings.py

```diff
@@ -21,6 +21,9 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
+# Path when redirecting to login
+LOGIN_URL = "/login/"
```

Comment:
This variable is defined in several places:
- `SORT/settings.py
- home/views.py`
- `survey/views.py`
it would be good to reuse this setting everywhere

Reply:
Fixed a while ago